endpoints/meme.py

```diff
@@ -14,9 +14,9 @@ class Meme(Endpoint):
     def generate(self, avatars, text, usernames):
         img = http.get_image(avatars[0]).convert('RGB')
         try:
-            font = self.assets.get_font(f'assets/fonts/{usernames[0].lower()}.ttf', size=48, )
```

Comment:
If this is a text-only command on Memer's side it won't attempt to resolve any provided arguments into a user so usernames[0] might never exist.

Reply:
Memegen is seperate from image commands in the bot. I'm simply re-using query param/json data that would be send to us, because these are already handled.

Reply:
In this case username1 would specify a font out of those supported, and username2 would specify a color. Memer should not attempt to parse a user for this command, but instead set these values manually when making it's request to imgen.